nox/virtualenv.py

```diff
@@ -305,6 +305,12 @@ def create(self) -> bool:
             return False
 
         cmd = [self.conda_cmd, "create", "--yes", "--prefix", self.location]
+        if self.conda_cmd == "micromamba" and not any(
+            v.startswith(("--channel=", "-c")) or v == "--channel"
+            for v in self.venv_params
+        ):
+            # Micromamba doesn't have any default channels
+            cmd.append("--channel=conda-forge")
```

Comment:
It might be worth mentioning this default in the docs as it is not quite the same as relying on PyPI (which doesn't re-package software from 3rd party authors as conda-forge does). Linking to the [conda-forge security considerations](https://conda-forge.org/docs/maintainer/knowledge_base/#security-considerations-for-conda-forge-builds) would be even better to inform users about that.